bower.json

```diff
@@ -1,8 +1,7 @@
 {
   "name": "o-topper",
   "main": [
-    "main.scss",
-    "main.js"
```

Comment:
Removing the JavaScript API from this component will mean that it is a breaking change and will need to have a migration guide added to the [migration file](https://github.com/Financial-Times/o-topper/blob/master/MIGRATION.md#migration-guide)

Reply:
Oh okay! I thought it could be a minor release because of what was discussed in this convo? See https://financialtimes.slack.com/archives/C02FU5ARJ/p1597829812214400?thread_ts=1597829441.212200&cid=C02FU5ARJ

Reply:
It can be made a minor if we keep the public JavaScript API. We can do that by including the new package as a dependency and having the o-topper `main.js` file re-export the new package. We did that with o-date, which may be useful to look at -- https://github.com/Financial-Times/o-date/blob/master/src/js/date.js#L289-L296

Reply:
OHHHHHH 🤦 Yes, that makes complete sense.